openslides_backend/presenter/presenter.py

```diff
@@ -120,16 +117,13 @@ def parse_presentations(self, payload: Payload) -> PresenterResponse:
         )
         response = []
         for presentation_blob in payload:
-            # user_id = presentation_blob["user_id"]
-            if presentation_blob["presentation"] == "dummy":
-                response.append(self.dummy_presentation())
+            Presenter = presenters.get(presentation_blob["presenter"])
+            if Presenter is not None:
+                p = Presenter()
```

Comment:
Maybe you have to inject the dependencies (database and permission) here too. Of course this is not required at the moment we can already prepare for it because I am sure we sill need then.

Reply:
ATM I do not understand the role of the `Presenter` (or if you will _pars pro toto_ the architecture) as such. For the moment I took the Presenter as a `ViewModel` presenting not necessarily retrieving the data required.